platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/event/generator/StandardGraphGenerator.java

```diff
@@ -201,11 +196,12 @@ private StandardGraphGenerator(final StandardGraphGenerator that, final long see
 
     private void initializeInternalConsensus() {
         consensus = new ConsensusImpl(platformContext, new NoOpConsensusMetrics(), addressBook);
-        inOrderLinker = new InOrderLinker(platformContext);
+        inOrderLinker = new ConsensusLinker(platformContext, new NodeId(0));
```

Comment:
probably obvious but why is this node(0), and why is it hardcoded?

Reply:
This field is only used for metrics, which are disabled when running with this framework. The value we pass into the constructor does not matter.